.travis.yml

```diff
@@ -78,7 +78,7 @@ before_install:
   # List all installed cargo packages.
   - cargo install --list
   # Install OSX requirements
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install libsodium rocksdb pkg-config; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install libsodium rocksdb pkg-config || brew install libsodium rocksdb pkg-config; fi
```

Comment:
Is the cause of this problem known? 

If it is a temporary workaround, please add a Jira ticket and a TODO here.

Reply:
I suppose we could update Homebrew on the virtual machine we run tests on. Corresponding ticket is created.